### PR TITLE
[dev] No PBI - Docs 0.12.0

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

File: docs/src/app/app.jsx [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
-import { Link } from 'react-router';
+import {
+    AppBar,
+    Button,
+    domUtils,
+    Icon,
+    Typography,
+} from 'react-cm-ui';
+import { Link, withRouter } from 'react-router';
+import _, {
```
Comment (War777) [inline docs/src/app/app.jsx:11]: Can we remove the underscore import? were only using a `_.debounce` in line 70.

File: docs/src/app/app.jsx [hunk 2]
```diff
@@ -1,20 +1,37 @@
...
-import { Button, domUtils, Icon } from 'react-cm-ui';
-import { Link } from 'react-router';
+import {
+    AppBar,
+    Button,
+    domUtils,
+    Icon,
+    Typography,
+} from 'react-cm-ui';
+import { Link, withRouter } from 'react-router';
```
Comment (War777) [inline docs/src/app/app.jsx:10]: Can we give it a line break?

File: docs/src/app/tabs.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import { browserHistory, withRouter } from 'react-router';
```
Comment (War777) [inline docs/src/app/tabs.jsx:1]: Can we give it a line break?

File: docs/src/app/tabs.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+import {
+    findIndex,
+    map,
+    remove,
+} from 'lodash';
+import { SectionalTabs } from 'react-cm-ui';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
```
Comment (War777) [inline docs/src/app/tabs.jsx:9]: Can we place each imported element in a new line after the `React` import?
```
import React, {
    useCallback,
    useEffect,
    useState,
} from 'react';
```

File: src/navigation/sectionalTabs/sectionalTabs.jsx [hunk 5]
```diff
@@ -424,7 +423,7 @@ class SectionalTabs extends Component {
         return reducedItems;
     }
 
-    getTabProps(tab) {
+    getTabProps(tab = {}) {
```
Comment (War777) [inline src/navigation/sectionalTabs/sectionalTabs.jsx:426]: Nice catch, I was getting an error in the client side :P thank you very much @morethanfire

File: src/navigation/sectionalTabs/sectionalTabs.jsx [hunk 6]
```diff
@@ -224,8 +224,7 @@ class SectionalTabs extends Component {
                 }),
                 ...(didPropsItemsChange && {
                     items,
-                    // let's select the first tab when there's a different items source
-                    selectedTabKey: get(items, '[0].key'),
+                    selectedTabKey,
```
Comment (War777) [inline src/navigation/sectionalTabs/sectionalTabs.jsx:227]: @morethanfire I see what you did in here, let me link this branch with the client so I can see how it behaves with the current Navigation progress, I'll get you updated hopefully tomorrow at morning at the scrum session.

Comment (War777) [reply src/navigation/sectionalTabs/sectionalTabs.jsx:227]: I already tested this changed, everything work fine in the side client with the current Navigation Changes, thanks @morethanfire

Comment (morethanfire) [reply src/navigation/sectionalTabs/sectionalTabs.jsx:227]: Nice!

File: docs/src/app/app.jsx (hunk 1 above)
Comment (morethanfire) [reply docs/src/app/app.jsx]: Fixed.

File: docs/src/app/tabs.jsx (hunk 3 above)
Comment (morethanfire) [reply docs/src/app/tabs.jsx]: Fixed.

File: docs/src/app/tabs.jsx (hunk 4 above)
Comment (morethanfire) [reply docs/src/app/tabs.jsx]: Fixed.

File: docs/src/layout/grid/grid.jsx [hunk 7]
```diff
@@ -676,8 +676,6 @@ export default class CollectionsGrid extends React.Component {
 
         return (
             <Main className="grid">
-                <TitleBar title="Grid" />
```
Comment (War777) [inline docs/src/layout/grid/grid.jsx:679]: `<TitleBar />` was removed in here but not in line 5 when importing from `react-cm-ui`.

File: docs/src/surfaces/modal/modal.jsx [hunk 8]
```diff
@@ -479,8 +479,6 @@ export default class ModulesModal extends React.Component {
 
         return (
             <Main page="headers">
-                <TitleBar title="Modal" />
```
Comment (War777) [inline docs/src/surfaces/modal/modal.jsx:482]: `<TitleBar />` was removed in here but not in line 3 when importing from `react-cm-ui`.

File: docs/src/surfaces/rail/rail.jsx [hunk 9]
```diff
@@ -64,8 +64,6 @@ export default class ElementsRail extends React.PureComponent {
 
         return (
             <Main page="rail">
-                <TitleBar title="Rail" />
```
Comment (War777) [inline docs/src/surfaces/rail/rail.jsx:67]: `<TitleBar />` was removed in here but not in line 6 (importing from `react-cm-ui`).

Comment (morethanfire) [reply docs/src/surfaces/rail/rail.jsx:67]: Fixed.

File: docs/src/surfaces/modal/modal.jsx (hunk 8 above)
Comment (morethanfire) [reply docs/src/surfaces/modal/modal.jsx:482]: Fixed.

File: docs/src/layout/grid/grid.jsx (hunk 7 above)
Comment (morethanfire) [reply docs/src/layout/grid/grid.jsx:679]: Fixed.

**Review by War777 — APPROVED** [review]

**Pull request merged**